core/server/data/migrations/versions/4.43/2022-04-01-10-13-add-post-newsletter-relation.js

```diff
@@ -27,14 +36,19 @@ module.exports = {
 
         logging.info(`Adding ${table}.${column} column`);
 
-        if (DatabaseInfo.isSQLite(knex)) {
-            await commands.addColumn(table, column, knex, columnDefinition);
-            return;
+        let sql = knex.schema.table(table, function (t) {
+            t.string(column, 24);
+        }).toSQL()[0].sql;
+
+        if (DatabaseInfo.isMySQL(knex)) {
+            sql += ', algorithm=copy';
         }
 
-        await knex.raw('alter table `posts` add column `newsletter_id` varchar(24) null, algorithm=copy;');
+        await knex.raw(sql);
 
-        await knex.raw('alter table `posts` add constraint `posts_newsletter_id_foreign` foreign key (`newsletter_id`) references `newsletters` (`id`);');
+        await knex.schema.alterTable(table, function (t) {
+            t.foreign(column).references('newsletters.id');
```

Comment:
Same as below except use the `addForeign`[^1] command

[^1]: https://github.com/TryGhost/Ghost/blob/main/core/server/data/schema/commands.js#L168